Add guideline B4c+ (BLD sight blocker timing) (#952)
thewca/wca-regulations: wca-regulations.md

- B4) Blindfolded phase:
    - B4a) The competitor dons the blindfold to start the blindfolded phase.
    - B4b) The competitor must not apply moves to the puzzle before they have fully donned the blindfold. Penalty: disqualification of the attempt (DNF).

>     - B4c) After the competitor has applied the first move to the puzzle, the judge must ensure that there is a sight blocker (e.g. a sheet of paper or cardboard) between the competitor's face and the puzzle.

Reviewer: A small nit: "after" technically applies to the entire solving period. We can use an alternative that makes it clear that the goal is to cover during as much of the solving period as possible.

```suggestion
    - B4c) As soon as the competitor applies the first move to the puzzle, the judge must ensure that there is a sight blocker (e.g. a sheet of paper or cardboard) between the competitor's face and the puzzle.
```
